twitter.py

```diff
@@ -0,0 +1,73 @@
+#Difficulties: How do I implement a custom comparator for the heapq used instead of priority queue module used in java as taught in class?
```

Comment:
You can define __gt__, __lt__ and __eq__ methods for the Tweet class. Please message me on Slack for more context, I'm unable to understand what you want to use the custom comparators for.